src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -49,6 +49,15 @@ def update
         redirect_back(fallback_location: root_path)
       end
 
+      def autocomplete_binaries
+        binaries = @image.find_binaries_by_name(params[:term])
+        autocomplete_result = []
+        binaries.each do |package, _|
+          autocomplete_result << {id: package, label: package, value: package}
```

Comment:
Because this is the format that jQueryUI autocomplete widget is using... It can change (I hope) and I prefer to have a model method that returns just the package name  + architectures not duplicated data that doesn't make any sense at that level (this is only a jQueryUI thing).

Reply:
Ok, I just don't like to iterate several times over the result but I'm fine with it!